catalogs/catalog-jdbc-doris/src/main/java/org/apache/gravitino/catalog/doris/utils/DorisUtils.java

```diff
@@ -202,11 +202,8 @@ public static Distribution extractDistributionInfoFromSql(String createTableSql)
                   .map(f -> f.substring(1, f.length() - 1))
                   .toArray(String[]::new);
 
-      // Default bucket number is 1.
-      int bucketNum = 1;
-      if (matcher.find(5)) {
-        bucketNum = Integer.valueOf(matcher.group(5));
-      }
+      // Default bucket number is 1. auto is -1.
```

Comment:
Should we also update the user doc to clarify this case?

BTW, how can a user create an auto-distributed Doris table by the Gravitino API?

Reply:
ok

Reply:
@mchades fixed , The Gravitino API can use `Distributions.auto(Strategy.HASH, NamedReference.field("score"));`.